src/code/z_actor.c

```diff
@@ -1991,28 +1991,40 @@ void Actor_InitContext(PlayState* play, ActorContext* actorCtx, ActorEntry* play
     func_8002FA60(play);
 }
 
-u32 D_80116068[ACTORCAT_MAX] = {
+u32 sCanIgnoreCategoryPlayerStateFlags1Masks[ACTORCAT_MAX] = {
```

Comment:
The name is kinda too verbose but I can't think of a better name either :sweat:

Reply:
Overall I think it'd be beneficial to use the term "freeze" over "ignore". Low level it does ignore the update, but it achieves the effect of actors freezing during various cutscene-like states or playing ocarina or something

Can try to stay more high level like sCategoryFreezeFlags and let a comment describe the intricacies instead of fitting it all in the name

Reply:
I agree "freeze" is better than "ignore", but it makes me think on actual freezing tho

Reply:
Pause?

Reply:
Note "freeze" is already used by `actor->freezeTimer` (so it would make sense to reuse it)

Reply:
Ohh, fair point, in that case using "freeze" here sounds good to me

Reply:
pause is an interesting idea, and could be discussed to be used instead of freeze. but yeah, would be good to keep everything in sync.
theres also this, which in my z_demo pr i named `haltAllActors`:
```c
                    if (!this->unk_11DE9) {
                        Actor_UpdateAll(this, &this->actorCtx);
                    }
```
so, that one should prob stay in sync with this terminology as well. whether halt is good or not for all of its uses, idk yet. need to put more thought (and it can prob happen outside of this pr)